### DISCUSSION TITLE
Need to disable the "Warning Potential Security Risk Ahead" screen

### DISCUSSION THREAD

**Discussion by Dave-0-0** [mozilla/policy-templates#1139] [Q&A]
Hello,

I posted this on that one Reddit site and it was suggested that I reach out or help over here. 

I am building some software to take screenshots of malicious websites using Firefox. When browsing to the malicious site, the "Warning Potential Security Risk Ahead" screen keeps getting the way. From researching, the following configuration should remove this, but it does not work. 

at /usr/lib/firefox/mozilla.cfg

` 
// IMPORTANT: Start your code on the 2nd line

lockPref("javascript.enabled", false);

// Bypass "Warning: Potential Security Risk Ahead" screens

lockPref("browser.ssl_override_behavior", 1);

lockPref("browser.xul.error_pages.enabled", true);

lockPref("browser.xul.error_pages.expert_bad_cert", true);

lockPref("security.certerrors.permanentOverride", true);

lockPref("security.certerrors.mitm.priming.enabled", false); `

then at /usr/lib/firefox/defaults/pref/autoconfig.js I have the following: 
` 
pref("general.config.filename", "mozilla.cfg");

pref("general.config.obscure_value", 0); // Use 0 to disable byte-shifting `

Does anyone know what is wrong with this? The browser does start with JavaScript disabled, so I know the files are being read. 

Thank you

**Comment by mkaply** [comment]
I checked with the security team and they don't think there is a safe way to do this with prefs.

Most of these prefs aren't a part of Firefox anymore.

Do you know the sites you'll be doing this with?

**Reply by ruser476** [reply]
Hi, I also have this issue. My reasoning is slightly different, as I install a lot of gateway devices. 

I've already set up a seperate system from which i'd be accessing the gateways from (and nothing else), is it at all possible to bypass any security warnings?

Thanks in advance.

**Reply by mkaply** [reply]
Are these known sites that you will be accessing repeatedly?

**Reply by ruser476** [reply]
Yeah. Always the same ones, at 192.168.x.x. (whatever subnet and ip they decide to use)
